persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/PatientMapper.xml

```diff
@@ -19,4 +19,57 @@
             </include>
         </if>
     </sql>
+
+    <sql id="from">
+        FROM patient
+        INNER JOIN cancer_study ON patient.CANCER_STUDY_ID = cancer_study.CANCER_STUDY_ID
+    </sql>
+
+    <sql id="where">
+        <where>
+            <if test="patientIds == null">
+                cancer_study.CANCER_STUDY_IDENTIFIER = #{studyIds[0]}
```

Comment:
so .. without patientIds, we will only look at the first study identifier in the list?  Intuitively, if a caller to the repository passes in a list of cancer studies, wouldn't they expect the union of all patients from all of the studies passed in?

perhaps the repository should throw an exception when called with a list of studies, but no patient Ids. (or at least, we should put comments by this repository functions in the interface explaining that such a call will ignore all but the first study)